moveit_ros/planning/planning_scene_monitor/src/trajectory_monitor.cpp

```diff
@@ -57,7 +59,7 @@ planning_scene_monitor::TrajectoryMonitor::~TrajectoryMonitor()
 void planning_scene_monitor::TrajectoryMonitor::setSamplingFrequency(double sampling_frequency)
 {
   if (sampling_frequency <= std::numeric_limits<double>::epsilon())
-    ROS_ERROR("The sampling frequency for trajectory states should be positive");
+    ROS_INFO_NAMED(LOGNAME, "Disabling trajectory recording");
   else
     sampling_frequency_ = sampling_frequency;
```

Comment:
I didn't test it yet, but it looks like you would want to set the member variable in the if-branch as well?

Reply:
You are absolutely right. I pushed this in a rush w/o prior testing...